services/prometheus-monitor.yml

```diff
@@ -585,9 +585,9 @@ metadata:
   annotations:
     volume.alpha.kubernetes.io/storage-class: default
 spec:
-  storageClassName: rook-ceph-block
+  storageClassName: ""
```

Comment:
I'm not sure this is what we want. I think we should be removing this line:

"PVCs don't necessarily have to request a class. A PVC with its storageClassName set equal to "" is always interpreted to be requesting a PV with no class, so it can only be bound to PVs with no class (no annotation or one set equal to ""). A PVC with no storageClassName is not quite the same and is treated differently by the cluster, depending on whether the DefaultStorageClass admission plugin is turned on."

https://kubernetes.io/docs/concepts/storage/persistent-volumes/